client/components/Comment.jsx

```diff
@@ -1,14 +1,31 @@
-/* eslint-disable react/prop-types */
 import { useParams } from 'react-router-dom';
 import styles from './Post.module.css';
 import useAuth from './useAuthContext';
 import axios from 'axios';
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
explain to me the usage of useCallback in your application, why is it useful ?

Reply:
umm to be honest its to shut up ESLint from complaining without disabling him.